src/Workspaces/CoreTestUtilities/OptionsCollection.cs

```diff
@@ -54,19 +52,22 @@ public void Add<T>(PerLanguageOption2<CodeStyleOption2<T>> option, T value)
         public void Add<T>(PerLanguageOption2<CodeStyleOption2<T>> option, T value, NotificationOption2 notification)
             => _options.Add(new OptionKey2(option, _languageName), new CodeStyleOption2<T>(value, notification));
 
+        [Obsolete("Use a strongly-typed overload instead.")]
```

Comment:
Why all the Obsolete attributes? Can we just delete these methods?

Reply:
Yep done